Engine/SerializerPlugins/ObjectSerializer.cs

```diff
@@ -592,7 +592,14 @@ public override bool Deserialize(XElement element, ITypeData t, Action<object> s
             }
             catch(Exception ex)
             {
-                Serializer.PushError(element, $"Object value was not read correctly.", ex);
+                if (ex is TargetInvocationException tarEx && tarEx.InnerException is ArgumentException arEx)
```

Comment:
Maybe we should just unwrap all TargetInvocationExceptions not only when the inner exception is ArgumentException?